hack/generated/apis/microsoft.resources/v1alpha1api20200601/resourcegroup_types.go

```diff
@@ -215,7 +215,7 @@ type ResourceGroupSpec struct {
 var _ genruntime.ARMTransformer = &ResourceGroupSpec{}
 var _ genruntime.ConvertibleSpec = &ResourceGroupSpec{}
 
-func (spec *ResourceGroupSpec) CreateEmptyARMValue() interface{} {
+func (spec *ResourceGroupSpec) CreateEmptyARMValue() genruntime.ARMResourceStatus {
 	return ResourceGroupSpecARM{}
```

Comment:
Should this not also return a ptr?

Reply:
Yes. The CI tests failed, and I fixed this with a later commit.